.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+# © 2024. TU Dortmund University,
+# Institute of Energy Systems, Energy Efficiency and Energy Economics,
+# Research group Distribution grid planning and operation
+#
+
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+    branches:
+      - main
+      - dev
+      - 'hotfix/*'
+      - 'rel/*'
+      - 'dependabot/*'
+  pull_request:
+    branches:
+      - main
+      - dev
+
+jobs:
+  build:
```

Comment:
Maybe you can find a better name for the job here. "Build and Test" or something like that?

Reply:
spaces arent allowed. I can try something like build/test